apps/foundation-demo/src/app/cart/components/cart-wrapper/cart-wrapper.component.spec.ts

```diff
@@ -58,10 +58,10 @@ class ProceedToCheckoutMock {}
 })
 class ContinueShoppingMock {}
 
-describe('TestCartAsyncWrapper', () => {
-  let component: TestCartAsyncWrapper;
-  let fixture: ComponentFixture<TestCartAsyncWrapper>;
-  let cartAsyncWrapperComponent: CartAsyncWrapperComponent;
+describe('TestCartWrapper', () => {
```

Comment:
This should be CartWrapper. Just the describe part, here. I must've messed this up when I wrote it originally.

Reply:
done